apps/api/src/app/step-schemas/usecases/placeholder-enrichment/email-editor-default-payload-creator-component.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { expect } from 'chai';
```

Comment:
Renaming this to `create-default-payload-for-email-editor.spec` reads much better.